server/repository/src/migrations/v2_05_00/item_direction_create_table.rs

```diff
@@ -0,0 +1,25 @@
+use crate::migrations::*;
+
+pub(crate) struct Migrate;
+
+impl MigrationFragment for Migrate {
+    fn identifier(&self) -> &'static str {
+        "item_direction_create_table"
+    }
+
+    fn migrate(&self, connection: &StorageConnection) -> anyhow::Result<()> {
+        sql!(
+            connection,
+            r#"
+            CREATE TABLE item_direction (
+                id TEXT NOT NULL PRIMARY KEY,
+                item_link_id TEXT NOT NULL REFERENCES item_link(id),
+                directions TEXT NOT NULL,
+                priority BIGINT NOT NULL
```

Comment:
I would have thought `INTEGER` would be ok, I guess big int is fine too.

Reply:
I had integer and diesel postgres was throwing a wobbly on i64 not receiving enough bytes